virtinst/guest.py

```diff
@@ -1108,6 +1117,8 @@ def defaults_to_pcie(self):
             return True
         if self.os.is_riscv_virt():
             return True
+        if self.os.is_loongarch64():
+            return True
```

Comment:
This was a very important spot that was missed the previous time around! Excellent catch :)

Things overall look good to me, but I recommend going one more time over all the spots where `self.os.is_*()` functions are used to make sure that nothing else is missed.

Reply:
Thanks!I'll check back.